doc/api/cli.md

```diff
@@ -1500,6 +1500,26 @@ changes:
 Configures the test runner to only execute top level tests that have the `only`
 option set.
 
+### `--shards`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Test suite shard to execute in a format of `<index>/<total>`, where
+
+`index` is a positive integer, index of divided parts
+`total` is a positive integer, total of divided part
+This command will divide all tests files into `total` equal parts, and will run only those that happen to be in an `index` part.
+
+For example, to split your tests suite into three parts, use this:
+
+```sh
+node --test --shard=1/3
+node --test --shard=2/3
+node --test --shard=3/3
```

Comment:
The flag is called `--shards` in code and in the title above, but written as `--shard` in the example code. It should be the same everywhere.

That said, `node --test --shards=1/3` would sound weird to me, as only a single shard is run but the name is in plural.
Also, the naming is inconsistent with other test runner flags that are prefixed with `test`.

My suggestion: `--test-shard`.

Reply:
Fixed the example, do you think that run option should be `shard` as well?

Reply:
it should be `--test-shard`, like all other test runner flags

Reply:
> do you think that the run option should be shard as well?

I would say yes.

Reply:
fixed, thanks